docker/checkout.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# CARMA packages checkout script
+
+set -ex
+
+git clone https://github.com/usdot-fhwa-stol/CARMAMsgs.git ~/src/CARMAMsgs
+git clone https://github.com/usdot-fhwa-stol/CARMADriverUtils.git ~/src/CARMADriverUtils
+sudo git clone https://github.com/usdot-fhwa-stol/autoware.ai /opt/autoware/src/autoware.ai/
```

Comment:
Is there a reason why you `sudo git clone ...` this in to `/opt/`, and then need to move it later instead of just cloning it directly into the home directory?